src/diffusers/__init__.py

```diff
@@ -80,6 +80,7 @@
             "AutoencoderTiny",
             "ConsistencyDecoderVAE",
             "ControlNetModel",
+            "PromptDiffusionControlNetModel",
```

Comment:
Can we maybe move this pipeline to the `research_projects` as it has a new model component too?